.circleci/docker/build.sh

```diff
@@ -165,7 +165,7 @@ case "$image" in
   pytorch-linux-xenial-py3-clang5-android-ndk-r19c)
     ANACONDA_PYTHON_VERSION=3.6
     CLANG_VERSION=5.0
-    CMAKE_VERSION=3.10.3
+    CMAKE_VERSION=3.13.5
```

Comment:
cc @malfet 

There is a bug in the Ninja generator where files considered inputs to the cmake command couldn't be generated by another build step. The fix requires 3.13 but 3.10.3 is still sufficient for other cmake generators e.g. makefiles
https://gitlab.kitware.com/cmake/cmake/-/issues/18584

Reply:
Please make sure to update https://github.com/pytorch/pytorch/blob/39ec0f321b2f5bd218dd1136ee08a951d876a33f/CMakeLists.txt#L1
as well